news/2024/2024-11-14-osu-world-cup-2024-round-of-16-recap.md

```diff
@@ -0,0 +1,218 @@
+
+China selected the remaining hidden map Escape for their last pick, but it backfired in the worst possible way, making Russia one point away from the win. For their final pick, Russia selected [DT1](https://osu.ppy.sh/beatmapsets/1126545#osu/2353845), and what ensued was a nail-biter. China held a narrow lead through much of the map, but into the ending Russia held a combo advantage as seconds counted down; a 200,000 point lead became 100,000, then 50,000, then 15,000 points as the final spinner appeared, but time ran out on Russia as China pulled off a break point to force the tiebreaker.
+
+Tiebreakers have long been a sore spot for Russia. They were eliminated from each of the last three World Cup iterations via tiebreaker (and in fact hadn't won a tiebreaker **since 2011**), so they had to break their streak of tiebreaker losses to continue in the winners bracket. As a battle raged on in [AJITAMA Adventure OST](https://osu.ppy.sh/beatmapsets/2274953#osu/4847546)'s storyboard, another one also raged between the two teams. When the dust finally settled, it was Russia that emerged victorious by a margin of just 102,726 points. With their spot in the Quarterfinals secured, the Russian roster could breathe a sigh of relief.
+
+### ::{ flag=ES }:: Spain vs. ::{ flag=FI }:: Finland
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116148023)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2298526307)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/Q3t9MojGc18?start=1132&end=1296" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Spain vs. Finland ended up being the star match in the loser's bracket last weekend — the only other match besides Russia vs. China that went down to the wire. The matchup was on an even playing field both seeding and skill-wise, which made for an extremely exciting match where only one pick brought a decisive victory with an 800k point difference for team Spain.
+
+Both teams were extremely confident in their strategies which ended up in a match where no breakpoints whatsoever occurred, the closest it got was during team Finland's [HD1](https://osu.ppy.sh/beatmapsets/2009101#osu/4179715) pick, ending with a razor-thin 71k difference between both teams. Some standout carry performances include ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899)'s 5x100 hidden FC on [FM2](https://osu.ppy.sh/beatmapsets/14309#osu/115384), and ::{ flag=ES }:: [creator](https://osu.ppy.sh/users/13962152)'s FC on [HD2](https://osu.ppy.sh/beatmapsets/1739433#osu/3555094) and near-FC on [HR2](https://osu.ppy.sh/beatmapsets/2274925#osu/4847498) that secured the tiebreaker for team Spain.
```

Comment:
Both teams were extremely confident in their strategies, which led to in a match where zero break points occurred; the closest to one was team Finland's [HD1](https://osu.ppy.sh/beatmapsets/2009101#osu/4179715) pick, ending with a razor-thin 71k difference between both teams. Some standout carry performances included ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899)'s 5x100 hidden FC on [FM2](https://osu.ppy.sh/beatmapsets/14309#osu/115384), and ::{ flag=ES }:: [creator](https://osu.ppy.sh/users/13962152)'s FC on [HD2](https://osu.ppy.sh/beatmapsets/1739433#osu/3555094) and near-FC on [HR2](https://osu.ppy.sh/beatmapsets/2274925#osu/4847498) that secured the tiebreaker for team Spain.

Reply:
done in https://github.com/ppy/osu-wiki/pull/12434/commits/0112ae1f855d9fd82b58ef29acfe52e7b437c01b